crates/ibc/src/core/ics04_channel/handler/chan_open_ack.rs

```diff
@@ -246,8 +244,7 @@ mod tests {
 
         let context = MockContext::default();
 
-        let msg_conn_init =
-            MsgConnectionOpenInit::try_from(get_dummy_raw_msg_conn_open_init(None)).unwrap();
+        let msg_conn_init = MsgConnectionOpenInit::new_dummy();
```

Comment:
Can we remove all changes in the `ics04_channel` module? This will cause a merge conflict with #430

Reply:
There still would be conflict, because of the `get_dummy_raw_msg_conn_open_*()` changes and it does not take `None` anymore, How if we merge this PR after #430?

Reply:
hmm, in that case let's just merge as is. We need to merge this first because the last step of #430 is to remove `impl Readers/Keepers for MockContext`, which needs this. They won't be big conflicts anyways